lib/urlutils.ts

```diff
@@ -1,9 +1,9 @@
 const deliveryDomain = process.env.NEXT_PUBLIC_MUX_BYO_DOMAIN || "mux.com";
 
-export function getStreamBaseUrl() {
+export function getStreamBaseUrl():string {
```

Comment:
nit (non-blocking): should be redundant, since TS should be able to infer type string.